lib/ex_oauth2_provider/access_grants/access_grant.ex

```diff
@@ -23,9 +23,9 @@ defmodule ExOauth2Provider.AccessGrants.AccessGrant do
   @doc false
   def attrs() do
     [
-      {:token, :string, null: false},
```

Comment:
Won't this mess up migrations?

Reply:
yup it did, reverting and gonna go with map.take on safe values